pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -56,6 +56,7 @@ impl<T: Config> From<RelayerMessageDecodingError> for Error<T> {
 pub mod pallet {
 	const BYTES_U32: usize = 4;
 	const BYTES_ACCOUNT_20: usize = 20;
+	const DEFAULT_WEIGHT_REF_TIME: u64 = 100_000_000;
```

Comment:
Please let me know if this value makes sense.

Reply:
THat is equal to the weight of a single write. I think it should be a magnitude higher and times 5 maybe.